pinot-common/src/test/java/com/linkedin/pinot/common/data/FieldSpecTest.java

```diff
@@ -192,6 +195,69 @@ public void testTimeFieldSpecConstructor() {
   }
 
   /**
+   * Test {@link TimeFieldSpec} constructors.
+   */
+  @Test
+  public void testDateTimeFieldSpecConstructor() {
```

Comment:
I would add negative test cases as well, with 0, -1, etc for numbers

Reply:
Refactored the tests to address the comments related to tests